### PR TITLE
SOE-705 added default_image_ft and set variable for fid to stanford_p…

### PR DESCRIPTION
**Pull request by boznik**
…erson_profile_picture

### CODE REVIEW

**Comment by boznik** [comment]
dependencies:
https://github.com/SU-SWS/stanford_install_tasks/pull/24
https://github.com/SU-SOE/stanford_sites_jumpstart_engineering/pull/33

**Comment by boznik** [comment]
making a few tweaks

**Comment by boznik** [comment]
not sure whether this should be tracked in strongarm. i left it out as it created override.

File: stanford_person.features.field_instance.inc [hunk 1]
```diff
@@ -1689,7 +1689,7 @@ Contact your site administrator to add or change entries. ',
     'required' => 0,
     'settings' => array(
       'alt_field' => 1,
-      'default_image' => 60,
```
Comment (sherakama) [inline stanford_person.features.field_instance.inc]: I thought I committed removing of the default_image fid...

**Comment by sherakama** [comment]
Looks good. Merge when you are ready.

**Pull request merged**